src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -46,13 +44,24 @@
   (let [top          (safe-area/get-top)
         selected-tab (reagent/atom (:id (first tabs-data)))]
     (fn []
-      (let [networks           (rf/sub [:wallet/network-details])
+      (let [tokens-loading?    (rf/sub [:wallet/tokens-loading?])
+            networks           (rf/sub [:wallet/network-details])
             account-cards-data (rf/sub [:wallet/account-cards-data])
             cards              (conj account-cards-data (new-account-card-data))]
         [rn/view {:style {:margin-top top :flex 1}}
          [common.top-nav/view]
          [rn/view {:style style/overview-container}
-          [quo/wallet-overview (temp/wallet-overview-state networks)]]
+          [quo/wallet-overview
```

Comment:
Maybe we can leave mock data in `temp` and pass `tokens-loading?` as an argument?

Reply:
Do we need this mock data?  Can't we get all real data at this point?

Reply:
@vkjr  - I will remove the keys which are not used at this moment (not displayed). That would reduce the key values. 

> Do we need this mock data? Can't we get all real data at this point?

@J-Son89 -  I'm starting on it 👍. https://github.com/status-im/status-mobile/issues/18196